src/api/emit.js

```diff
@@ -1,25 +1,8 @@
 import createElement from '../native/create-element';
-import createEvent from '../native/create-event';
+import createCustomEvent from '../util/create-custom-event';
```

Comment:
Is there any reasoning for separating this out and not just using `emit()` directly? I guess you don't need all the behaviour that emit provides. More curious that anything else.